sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/ItemsPartitionReader.scala

```diff
@@ -237,7 +233,6 @@ private case class ItemsPartitionReader
       )
 
       queryOptions.setDedicatedGatewayRequestOptions(readConfig.dedicatedGatewayRequestOptions)
-      queryOptions.setCosmosEndToEndOperationLatencyPolicyConfig(endToEndTimeoutPolicy)
```

Comment:
does it only impact the cases where empty pages being returned and SDK need to draining until getting something back? 

or the query plan? 

and do we need to remove the timeout gating in the `TransientIOErrorsRetryingIterator` as well?